commands/app/migrate.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  addAccountOptions,
+  addConfigOptions,
+  addUseEnvironmentOptions,
+} from '../../lib/commonOpts';
+import {
+  trackCommandMetadataUsage,
+  trackCommandUsage,
+} from '../../lib/usageTracking';
+import { i18n } from '../../lib/lang';
+import { ApiErrorContext, logError } from '../../lib/errorHandlers';
+import { EXIT_CODES } from '../../lib/enums/exitCodes';
+import { getAccountConfig } from '@hubspot/local-dev-lib/config';
+import { ArgumentsCamelCase, Argv, CommandModule } from 'yargs';
+import { MigrateAppOptions } from '../../types/Yargs';
+import { migrateApp2023_2, migrateApp2025_2 } from '../../lib/app/migrate';
+import { PLATFORM_VERSIONS } from '@hubspot/local-dev-lib/constants/projects';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { uiBetaTag, uiLink } from '../../lib/ui';
+
+const { v2023_2, v2025_2, unstable } = PLATFORM_VERSIONS;
+export const validMigrationTargets = [v2023_2, v2025_2, unstable];
+
+const command = 'migrate';
+const describe = undefined; // uiBetaTag(i18n(`commands.project.subcommands.migrateApp.header.text.describe`), false);
+
+export async function handler(options: ArgumentsCamelCase<MigrateAppOptions>) {
+  const { derivedAccountId, platformVersion } = options;
+  await trackCommandUsage('migrate-app', {}, derivedAccountId);
+  const accountConfig = getAccountConfig(derivedAccountId);
+
+  if (!accountConfig) {
+    logger.error(
+      i18n(`commands.project.subcommands.migrateApp.errors.noAccountConfig`)
+    );
+    return process.exit(EXIT_CODES.ERROR);
+  }
+
+  logger.log('');
+  logger.log(
+    uiBetaTag(
+      i18n(`commands.project.subcommands.migrateApp.header.text`),
```

Comment:
Can you shorten this with an `i18nKey` defined above?

Reply:
We are actively removing the string interpolation with the `i18nKeys` because they make it harder to determine which string we are using and having the full string will make the migration to the native JS object easier.